examples/rl_gsm8k/orchestrate_rl.py

```diff
@@ -315,9 +315,21 @@ def main(cfg: DictConfig):
     setup_logging(exp_path)
     logger.info(f"Current dir: {os.getcwd()}, output dir: {cfg.output_dir}")
     cfg.finetune.wandb_id = exp_path.name
-    run = init_wandb(cfg, exp_path, flatten_dict_config(cfg))
-    if run is None:
-        raise ValueError("Failed to initialize wandb run")
+    
+    # Initialize wandb and handle failure gracefully
+    try:
+        run = init_wandb(cfg, exp_path, flatten_dict_config(cfg))
```

Comment:
Does it often happen for you that wandb fails to initialize? What's the point of running a job without wandb logs for us?

Reply:
Not much point. But we can only initialise it on one node so the other one has to set the wandb instance to None to skip it gracefully.